src/Services/ProcedureService.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+/**
+ * ProcedureService
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Yash Bothra <yashrajbothra786gmail.com>
+ * @copyright Copyright (c) 2020 Yash Bothra <yashrajbothra786gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Services;
+
+use OpenEMR\Common\Uuid\UuidRegistry;
+use OpenEMR\Validators\BaseValidator;
+use OpenEMR\Validators\ProcessingResult;
+
+class ProcedureService extends BaseService
+{
+
+    private const PROCEDURE_TABLE = "procedure_order";
```

Comment:
to be consistent (since placing this stuff in constants), would also do:
```
private const PROCEDURE_TABLE_ID = "procedure_order_id";
```
and then use it below in the uuidRegistry call